v3/views/readme.pg

```diff
@@ -0,0 +1,5 @@
+node :readme do
+  node track_id: problem.track_id
+  node slug: problem.slug
+  node readme: problem.readme
```

Comment:
Let's also include the language name (`track.language`) and the problem name `problem.name`.